Libs/MRML/Core/vtkMRMLFolderDisplayNode.cxx

```diff
@@ -198,6 +199,13 @@ void vtkMRMLFolderDisplayNode::ChildDisplayNodesModified()
   // Get items in branch
   std::vector<vtkIdType> childItemIDs;
   shNode->GetItemChildren(folderItemId, childItemIDs, true);
+
+  bool batchProcessing = (childItemIDs.size() > 10);
```

Comment:
What would be the cons of always do batch processing ?

Reply:
There is some overhead for doing batch processing. This number is supposed to compensate for that.